README.md

```diff
@@ -301,7 +301,7 @@ Sometimes the plugin may crash unexpectedly, you may notice:
 
 - you will see a response in red text below the chat window displaying ` slash command with trigger of  '/(name)' not found,`  - If you check your log file, look for messages that refer to "plugins" and "health check fail", "ExecuteCommand" etc. 
 
-If you encopunter these types of issues - you can set `LogSettings.FileLevel` to `DEBUG` in your config.json settings.   This will enable debug logging and give more verbose error events in the system log. Then try re-enabling the plugin in the system-console. These log results may be requested by others in the forum or by our support team.
+If you encopunter these types of issues - you can set `LogSettings.FileLevel` to `DEBUG` in your config.json settings.   This will enable debug logging and give more verbose error events in the system log. Then try re-enabling the plugin in the system-console. These log results may be requested by others in the forum or by our support team.  **Note** If you have a site with high volumes of activity, this setting can cause Log files to expand substantially.  Keep an eye on your server logs, or only enable in development environments.
```

Comment:
```suggestion
If you encounter these types of issues - you can set `LogSettings.FileLevel` to `DEBUG` in your config.json settings.   This will enable debug logging and give more verbose error events in the system log. Then try re-enabling the plugin in the system-console. These log results may be requested by others in the forum or by our support team.  **Note** If you have a site with high volumes of activity, this setting can cause Log files to expand substantially.  Keep an eye on your server logs, or only enable in development environments.
```